### ISSUE TITLE
"drop measurement" hangs

### ISSUE THREAD

**Issue by AdamJacobMuller** [influxdata/influxdb#8504]
### Bug report

__System info:__ Ubuntu 14.04, Kernel 3.16.0-46, influxdb 1.2.2-1 (from official repo)

__Steps to reproduce:__

1. "drop measurement process_virtual_memory_bytes"
2. wait
3. wait more
4. file bug report while still waiting (>2h so far)

__Expected behavior:__ measurement to be dropped

__Actual behavior:__ nothing... still waiting

Total influxdb database is not that large (~100GB), this series is probably not even 1% of that so this seems like a relatively tiny amount of data.

I've seen very similar issues with "drop series from process_virtual_memory_bytes" and "drop series where ...".

"debug/pprof/all?cpu=true" returns a 404 "Unknown profile: all" ([http] pprof-enabled = true in influxdb.conf)

vars: https://gist.github.com/AdamJacobMuller/bb44181c09f8e72c2b189b7293ed27c1

I know this isn't enough information to fix, but, I'm more than open to gathering more information, just let me know what you need. In particular, any ideas how to get pprof data?

**Comment by AdamJacobMuller** [comment]
influxdb eventually started rejecting writes, in an increasing percentage until:

> [I] 2017-06-17T06:01:59Z write failed for shard 49: engine: cache-max-memory-size exceeded: (1048690449/1048576000) service=write

a lot of these appeared, then memory usage eventually started to balloon until influxdb consumed all the memory in the system (24G) and the system OOM-killed influxdb. its doing whatever it does on crash recovery for ~15m now.

**Comment by AdamJacobMuller** [comment]
after recovery and an upgrade to 1.2.4:

> > drop measurement process_virtual_memory_bytes;
> ERR: measurement not found: process_virtual_memory_bytes

**Comment by AdamJacobMuller** [comment]
after a few more crashes, I left the system in a state of not accepting any writes for ~12 hours, it did a lot of compaction and then I was able to run `drop measurement` in a normal timeframe (a few seconds per measurement), restored writes and everything seems fine, besides ~20 hours of missing data

**Comment by AdamJacobMuller** [comment]
also relevant, after restoring writes, I can confirm that there are no more writes to the measurements I was attempting to drop.

**Comment by jwilder** [comment]
This should be fixed in 1.3. See #8204.

**Comment by kmonsoor** [comment]
Possibly, this issue is still there.

Today, after I updated from `1.2.4` to `1.3.2` and restarted `InfluxDB` service, things were going smooth. Then, I installed Chronograf on the same instance.
To make some graph, I noticed the measurement list is extremely slow to load. 
I immediately stopped the `chronograph` service. Then, started to cleanup some old and unnecessary measurements. First, couple ones were done almost instantly. They something went haywire.
Now, it is completed after ~1 hour, consuming 1.5GB+ memory.

Here's the output from `show stats`: https://gist.github.com/kmonsoor/cba814077959bdd237e7ebb3b968ad2b

Here's current `htop` screenshot. Notice the top item.

![influx-drop-series](https://user-images.githubusercontent.com/1219418/29286503-17193522-8154-11e7-8fe8-9bca1c12d2be.PNG)
